dask_expr/io/parquet.py

```diff
@@ -516,7 +515,7 @@ def pyarrow_type_mapper(pyarrow_dtype):
         type_mappers.append(user_types_mapper)
 
     # next in priority is converting strings
-    if convert_string:
+    if pyarrow_strings_enabled():
```

Comment:
This is problematic, this would pull whatever is set on the cluster at that point in time. dask/dask preserves whatever is set when generating the collection. The fsspec implementation does this too but it's hidden very deep inside the legacy implementation

Reply:
good point. that's easily fixed/reverted